src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingRunnable.java

```diff
@@ -42,20 +43,25 @@ public SyncthingRunnable(Context context, String command) {
 
     @Override
     public void run() {
-        SharedPreferences pm = PreferenceManager.getDefaultSharedPreferences(mContext);
+        SharedPreferences sp = PreferenceManager.getDefaultSharedPreferences(mContext);
         DataOutputStream dos = null;
         int ret = 1;
         Process process = null;
         try {
             // Loop to handle syncthing restarts (these always have an error code of 3).
             do {
-                process = Runtime.getRuntime().exec("sh");
+                ProcessBuilder pb = new ProcessBuilder("sh");
```

Comment:
@Zillode I tried to pass `mCommand` here, but that broke config generation. Maybe you can get this working in #283?